mix.exs

```diff
@@ -57,7 +57,7 @@ defmodule CredoMox.MixProject do
   # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
-      {:credo, "~> 1.7", only: [:dev, :test], runtime: false},
+      {:credo, "~> 1.7", only: [:dev, :test]},
```

Comment:
I think we actually need to remove the `[:dev, :test]` here. Otherwise we get a warning when running `mix credo`:
`** (config) Ignoring an undefined check: CredoMox.Checks.UnverifiedMox`

This is happening because the `Code.ensure_loaded?(Credo.Check)` is returning false since when `:credo_mox` is used as a dependency it is compiled with `MIX_ENV=prod` so it doesn't have any dependency on Credo.

Removing `[:dev, :test]` won't have any appreciable impacts on consumers of this package as long as they add credo_mox with `{:credo_mox, "~> 0.1", only: [:dev, :test], runtime: false}`